src/Microsoft.DotNet.Build.Tasks/CleanupVSTSAgent.cs

```diff
@@ -29,11 +29,14 @@ public class CleanupVSTSAgent : Microsoft.Build.Utilities.Task
 
         public int MaximumWorkspacesToClean { get; set; } = 8;
 
+        public bool AttemptMaxPathFolderRemoval { get; set; } = true;
+
         public int? SleepTimeInMilliseconds { get; set; }
         public ITaskItem[] ProcessNamesToKill { get; set; }
 
         private static readonly int s_DefaultRetries = 3;
         private static readonly int s_DefaultSleepTime = 2000;
+        private DateTime timerStarted;
```

Comment:
according to https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md, this should be `_timeStarted`